extras/profile-and-settings.rst

```diff
@@ -47,6 +47,15 @@ Profile Settings
 
    Consult the `admin documentation <https://admin-docs.zammad.org/en/latest/manage-trigger.html#other-notifications>`_
    for details on how to customize these email notifications.
+   
+   The following *Limit Groups* box allows you to disable notifications for the unchecked 
+   groups. If a group is not checked, a group ticket will not trigger a notification. 
+   You will still get desktop notifications, and email notifications for tickets directly assigned to you.
+
+   .. figure:: /images/extras/profile-limit-notifications-to-specific-groups.png
```

Comment:
To fit the naming scheme of other images in the documentation, the file should be named `profile-and-settings-notifications-limit-groups.png`.

Reply:
That's fine by me!

Reply:
ugh sorry my bad - this commit https://github.com/zammad/zammad-user-documentation/pull/22/commits/d08ba2716abf029128c63f0ef98ee5e73f8027a7 will fix the naming of the image :)